pkg/virtual/framework/dynamic/apiserver/apiserver.go

```diff
@@ -104,9 +104,9 @@ func (c completedConfig) New(virtualWorkspaceName string, delegationTarget gener
 
 	director := genericServer.Handler.Director
 	genericServer.Handler.Director = http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
-		vwName, err := virtualcontext.VirtualWorkspaceNameFrom(r.Context())
-		if err != nil {
-			klog.Error(err)
+		vwName, vwNameExists := virtualcontext.VirtualWorkspaceNameFrom(r.Context())
```

Comment:
convention is `..., ok := ...` or `found`

Reply:
I'll use `found` then

Reply:
Fixed in https://github.com/kcp-dev/kcp/pull/1185/commits/52fd3afddd66d1f586bc2d1fad3f66e4c3215d13